clients/ts/signalr/tests/HttpConnection.test.ts

```diff
@@ -643,17 +643,19 @@ describe("HttpConnection", () => {
         it("uses EventSource constructor from options if provided", async () => {
             let eventSourceConstructorCalled: boolean = false;
 
-            const customEventSourceType = class EventSource {
-                constructor(url: string, eventSourceInitDict?: EventSourceInit) {
+            class TestEventSource {
```

Comment:
These were mostly a style change rather than the linter. Since the class can just be local to the function and have it's own name, there's no need to assign it to a variable.